product_taxonomy-etl.gemspec

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |spec|
+  spec.name          = "product_taxonomy-etl"
+  
+  # Extract version from git tag (e.g., etl-v1.2.0 -> 1.2.0)
+  # Falls back to "0.0.0" if no matching tag is found
+  spec.version       = `git describe --tags --match "etl-v*" 2>/dev/null`.strip.sub(/^etl-v/, '') || "0.0.0"
+  
+  spec.authors       = ["Shopify"]
+  spec.email         = ["taxonomy@shopify.com"]
+
+  spec.summary       = "ETL utilities for Shopify Product Taxonomy"
```

Comment:
We should expand the "ETL" acronym here and explain a bit more about what it does so that someone without context can understand.